### ISSUE TITLE
Regarding the prim.niggli.json

### ISSUE THREAD

**Issue by darjaved** [prisms-center/CASMcode#356]
I have done my calculation on my own server with the enumeration part done my Pymatgen. When i use the prim file for initializing the casm project it gives me the prim.niggli.json file, should i use this suggested file as my prim structurem now. Any suggestions? why i actually visualized the suggested prim the distance between the atoms is lesser than what i have actually. so, can i start the project by force by using my own prim?

**Comment by xivh** [comment]
I don't think it will matter which one you use. If you think that there is something wrong with the atom distances, you can try importing unrelaxed structures into a test project to see what the mapping scores are (inside reports folder). They should be 0 if the unrelaxed structure matches your prim.

**Comment by darjaved** [comment]
Is matching enough? having a wrong atomic distance as in the suggested prim file can lead to wrong CE fit.

**Comment by xivh** [comment]
If the import has 0 score then the distances should match. Can you upload your niggli prim and your expected prim? Is it just in a rotated or translated supercell?
